### PR TITLE
Adding cccp.yml to origin-source container.

### PR DESCRIPTION
**Pull request by mohammedzee1000**
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

**Comment by kargakis** [comment]
@mohammedzee1000 can you explain why this is needed? Sorry, I am not familiar with the centos processes.

**Comment by mohammedzee1000** [comment]
The cccp yaml file is needed to be present in the repository to allow the CentOS Container Pipeline to build this image (https://github.com/CentOS/container-pipeline-service).

Based on earlier discussions of course, CentOS Containers were to pick up rpms published by the CentOS PaaS Sig.

Although it might be possible to build the rpms through the pipeline pre-build step, by providing a pre-build script that does that. I don't know how recommended that will be.

@kbsingh @bamachrn, comments?

Container index entries:
- https://github.com/CentOS/container-index/blob/master/index.d/openshift.yaml#L4-L14
- https://github.com/CentOS/container-index/blob/master/index.d/openshift.yaml#L265-L275

File: images/source/Dockerfile.pipeline [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#
+# This image adds the repository link for Origin
+# RPMs to allow downstream images to install the
+# components using `yum'.
+#
+# The standard name for this image is openshift/origin-source
+#
+FROM centos:7
+
+RUN yum -y install centos-release-openshift-origin39 && yum clean all
```
Comment (kargakis) [inline images/source/Dockerfile.pipeline]: I am wondering whether this is the right place to be installing origin. source is supposed to setup the source for downstream images (see comment above).

Comment (mohammedzee1000) [reply images/source/Dockerfile.pipeline]: And that is what is happening here. This rpm configures the yum repositories for rpm based install, so later images can do something like yum install 


```
$ ls /etc/yum.repos.d/       
CentOS-Base.repo  CentOS-Debuginfo.repo  CentOS-Sources.repo  CentOS-fasttrack.repo
CentOS-CR.repo    CentOS-Media.repo      CentOS-Vault.repo
$  yum search origin
Loaded plugins: fastestmirror, ovl
base                                                                              | 3.6 kB  00:00:00     
extras                                                                            | 3.4 kB  00:00:00     
updates                                                                           | 3.4 kB  00:00:00     
(1/4): extras/7/x86_64/primary_db                                                 | 173 kB  00:00:00     
(2/4): base/7/x86_64/group_gz                                                     | 166 kB  00:00:00     
(3/4): base/7/x86_64/primary_db                                                   | 5.9 MB  00:00:01     
(4/4): updates/7/x86_64/primary_db                                                | 4.3 MB  00:00:06     
Determining fastest mirrors
 * base: centos.excellmedia.net
 * extras: centos.excellmedia.net
 * updates: centos.excellmedia.net
========================================== N/S matched: origin ==========================================
centos-release-openshift-origin13.noarch : Yum configuration for OpenShift Origin 1.3 packages
centos-release-openshift-origin14.noarch : Yum configuration for OpenShift Origin 1.4 packages
centos-release-openshift-origin15.noarch : Yum configuration for OpenShift Origin 1.5 packages
centos-release-openshift-origin36.noarch : Yum configuration for OpenShift Origin 3.6 packages
centos-release-openshift-origin37.noarch : Yum configuration for OpenShift Origin 3.7 packages
centos-release-openshift-origin39.noarch : Yum configuration for OpenShift Origin 3.9 packages
google-noto-sans-canadian-aboriginal-fonts.noarch : Sans Canadian Aboriginal font
centos-release-openshift-origin.noarch : Common release file to establish shared metadata for CentOS PaaS
                                       : SIG
ksh.x86_64 : The Original ATT Korn Shell
texlive-tetex.noarch : scripts and files originally written for or included in teTeX
$ ls /etc/yum.repos.d/
CentOS-Base.repo  CentOS-Debuginfo.repo  CentOS-OpenShift-Origin39.repo  CentOS-Vault.repo
CentOS-CR.repo    CentOS-Media.repo      CentOS-Sources.repo             CentOS-fasttrack.repo

$ yum search origin
Loaded plugins: fastestmirror, ovl
centos-openshift-origin39                                                         | 2.9 kB  00:00:00     
centos-openshift-origin39/primary_db                                              |  83 kB  00:00:00     
Loading mirror speeds from cached hostfile
 * base: centos.excellmedia.net
 * extras: centos.excellmedia.net
 * updates: centos.excellmedia.net
========================================== N/S matched: origin ==========================================
centos-release-openshift-origin13.noarch : Yum configuration for OpenShift Origin 1.3 packages
centos-release-openshift-origin14.noarch : Yum configuration for OpenShift Origin 1.4 packages
centos-release-openshift-origin15.noarch : Yum configuration for OpenShift Origin 1.5 packages
centos-release-openshift-origin36.noarch : Yum configuration for OpenShift Origin 3.6 packages
centos-release-openshift-origin37.noarch : Yum configuration for OpenShift Origin 3.7 packages
centos-release-openshift-origin39.noarch : Yum configuration for OpenShift Origin 3.9 packages
google-noto-sans-canadian-aboriginal-fonts.noarch : Sans Canadian Aboriginal font
origin-clients.x86_64 : Origin Client binaries for Linux
origin-clients-redistributable.x86_64 : Origin Client binaries for Linux, Mac OSX, and Windows
origin-cluster-capacity.x86_64 : Origin Cluster Capacity Analysis Tool
origin-master.x86_64 : Origin Master
origin-node.x86_64 : Origin Node
origin-pod.x86_64 : Origin Pod
origin-sdn-ovs.x86_64 : Origin SDN Plugin for Open vSwitch
origin-service-catalog.x86_64 : Origin Service Catalog
origin-tests.x86_64 : Origin Test Suite
centos-release-openshift-origin.noarch : Common release file to establish shared metadata for CentOS PaaS
                                       : SIG
ksh.x86_64 : The Original ATT Korn Shell
origin.x86_64 : Open Source Container Management by Red Hat
origin-docker-excluder.noarch : Exclude docker packages from updates
origin-excluder.noarch : Exclude openshift packages from updates
origin-federation-services.x86_64 : %{produce_name} Federation Services
origin-template-service-broker.x86_64 : Template Service Broker
texlive-tetex.noarch : scripts and files originally written for or included in teTeX

  Name and summary matches only, use "search all" for everything.

```
@stevekuznetsov correct me if im wrong

**Pull request closed without merging**